specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2021-05-17-preview/privateLinkScopes.json

```diff
@@ -800,6 +800,14 @@
           "readOnly": true,
           "description": "The Guid id of the private link scope.",
           "type": "string"
+        },
+        "privateEndpointConnections": {
```

Comment:
I see privateEndpointConnections array defined here contains the list of connections and the details. Few questions...

1. How many privateEndpointConnections can exist? Would all the connections details fit in a single response? Note that you cannot do paging just for the array content. 
2. Also you will run into RBAC issues if you return the entire privateEndpointConnection response in privateLinkScope GET. Refer https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc019-rbac-violation-information-leak-when-resources-full-content-of-other-types-are-included-in-the-response

I would recommend to return just an array of privateEndpointConnection IDs.

Reply:
1. Single digits. Most likely 1, but we wanted to make sure we had support built in for >1
2. The privateEndpointConnection data isn't really a different resource type, but just data on a privateLinkScope. it also isn't something that should have different RBAC rules associated with it. This information is needed for policy checks, so we need it to be available here.

Reply:
adding on to 2., the data we're returning here isn't the underlying private endpoint resource itself either. It's just some information about the private link scope's relationship with that private endpoint, plus the id of the private endpoint. I don't think there should be any information leak issues here.

Reply:
@dkirby017 Sounds good then. Signing off by adding 'ARMSignedOff' label.